### ISSUE TITLE
TS2322: Type 'JQuery<Element>' is not assignable to type 'JQuery<HTMLElement>'.

### ISSUE THREAD

**Issue by TheLifeofO** [akeneo/pim-community-dev#11995]
When running `yarn webpack` i receive the following error

```
tsl] ERROR in /srv/pim/web/bundles/pimdatagrid/js/datafilter/filters-column.ts(285,5)
      TS2322: Type 'JQuery<Element>' is not assignable to type 'JQuery<HTMLElement>'.
  Type 'Element' is missing the following properties from type 'HTMLElement': accessKey, accessKeyLabel, autocapitalize, dir, and 109 more.
```

Looks like this property
```
public filterList: JQuery<HTMLElement>;
```

Needs to be typed to `JQuery<Element>`

**Comment by Rashguir** [comment]
i have the same issue when installing akeneo 3.2

**Comment by robingchan** [comment]
**this is just a temporary fix**

temporary fix is open this file:
web/bundles/pimdatagrid/js/datafilter/filters-column.ts

change line ~25 from this:
public filterList: JQuery<HTMLElement>;

to this:
public filterList: JQuery<Element>;

then run:
yarn run webpack

**Comment by simoncarre** [comment]
Same issue here with v3.2.46

**Comment by jmleroux** [comment]
Hey @robingchan ,

We can fix it, but as we are open source, a PR would be also very welcome :wink:

**Comment by tseho** [comment]
2 days ago: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/a6043265237484b65dc219e7b4b900377c927328

**Comment by tseho** [comment]
JQuery.find signature just changed :/

**Comment by jmleroux** [comment]
Nice catch Quentin.

**Comment by jmleroux** [comment]
Bad catch Jquery :facepalm:

**Comment by jmleroux** [comment]
Merged in 3.2, it will be available in next v3.2.52 tag.

**Comment by tseho** [comment]
Thank you all, for taking the time to report the issue.
